paper/example-modular-augur-workflows.dot

```diff
@@ -0,0 +1,66 @@
+digraph G {
+    graph[margin=0];
+
+    subgraph cluster_zika_snakemake_dag {
+        graph[bgcolor=white, color=white, margin=0, center=true, label="A                          ", labeljus=left, labelloc=top, fontsize=20, size=4.25];
+        node[shape=box, style=rounded, fontname=sans, fontsize=12, penwidth=2];
+        edge[penwidth=2, color=grey];
+        z1[label = "export", color = "0.27 0.6 0.85", style="rounded"];
+        z2[label = "refine", color = "0.60 0.6 0.85", style="rounded"];
+        z3[label = "traits", color = "0.07 0.6 0.85", style="rounded"];
+        z4[label = "ancestral", color = "0.40 0.6 0.85", style="rounded"];
+        z5[label = "translate", color = "0.33 0.6 0.85", style="rounded"];
+        z6[label = "tree", color = "0.13 0.6 0.85", style="rounded"];
+        z7[label = "align", color = "0.00 0.6 0.85", style="rounded"];
+        z8[label = "filter", color = "0.53 0.6 0.85", style="rounded"];
+        z2 -> z1
+        z3 -> z1
+        z4 -> z1
+        z5 -> z1
+        z6 -> z2
+        z7 -> z2
+        z2 -> z3
+        z2 -> z4
+        z7 -> z4
+        z2 -> z5
+        z4 -> z5
+        z7 -> z6
+        z8 -> z7
+    }
+
+    subgraph cluster_tb_snakemake_dag {
+        graph[bgcolor=white, color=white, margin=0, center=true, label="B                          ", labeljus=left, labelloc=top, fontsize=20, size=4.25];
+        node[shape=box, style=rounded, fontname=sans, fontsize=12, penwidth=2];
+        edge[penwidth=2, color=grey];
+        t1[label = "export", color = "0.22 0.6 0.85", style="rounded"];
+        t2[label = "refine", color = "0.28 0.6 0.85", style="rounded"];
+        t3[label = "traits", color = "0.00 0.6 0.85", style="rounded"];
+        t4[label = "ancestral", color = "0.33 0.6 0.85", style="rounded"];
+        t5[label = "translate", color = "0.44 0.6 0.85", style="rounded"];
+        t6[label = "seqtraits", color = "0.56 0.6 0.85", style="rounded"];
+        t7[label = "clades", color = "0.06 0.6 0.85", style="rounded"];
+        t8[label = "tree", color = "0.17 0.6 0.85", style="rounded"];
+        t9[label = "mask", color = "0.39 0.6 0.85", style="rounded"];
+        t10[label = "filter", color = "0.50 0.6 0.85", style="rounded"];
+        t2 -> t1
+        t3 -> t1
+        t4 -> t1
+        t5 -> t1
+        t6 -> t1
+        t7 -> t1
+        t8 -> t2
+        t9 -> t2
+        t2 -> t3
+        t2 -> t4
+        t9 -> t4
+        t2 -> t5
+        t4 -> t5
+        t4 -> t6
+        t5 -> t6
+        t2 -> t7
+        t5 -> t7
+        t4 -> t7
+        t9 -> t8
+        t10 -> t9
+    }
```

Comment:
Here are two potential additions which I think present the versatility of augur (in addition to the modularity). The aesthetics need work, but I think the final image should be put through sketch or similar anyway. 

```

    subgraph existing_alignment {
        graph[bgcolor=white, color=white, margin=0, center=true, label="Existing alignment", labeljus=left, labelloc=top, fontsize=20, size=4.25];
        node[shape=box, style=rounded, fontname=sans, fontsize=12, penwidth=2];
        edge[penwidth=2, color=grey];
        e1[label = "Existing alignment", color = "0.50 0.6 0.85", style="rounded"];
        e2[label = "New Sequences", color = "0.50 0.6 0.85", style="rounded"];
        e3[label = "Existing metadata", color = "0.50 0.6 0.85", style="rounded"];
        e4[label = "New Metadata", color = "0.50 0.6 0.85", style="rounded"];
        e5[label = "Custom metadata parse script", color = "0.50 0.6 0.85", style="rounded"];
        e6[label = "augur align --existing-alignment", color = "0.50 0.6 0.85", style="rounded"];
        e7[label = "[rest of pipeline]", color = "0.50 0.6 0.85", style="rounded"];
        e1 -> e6
        e2 -> e6
        e3 -> e5
        e4 -> e5
        e5 -> e7
        e6 -> e7
    }

    subgraph beast{
        graph[bgcolor=white, color=white, margin=0, center=true, label="BEAST", labeljus=left, labelloc=top, fontsize=20, size=4.25];
        node[shape=box, style=rounded, fontname=sans, fontsize=12, penwidth=2];
        edge[penwidth=2, color=grey];
        b1[label = "BEAST analysis", color = "0.22 0.6 0.85", style="rounded"];
        b2[label = "augur import beast", color = "0.22 0.6 0.85", style="rounded"];
        b3[label = "augur traits (DTA)", color = "0.22 0.6 0.85", style="rounded"];
        b4[label = "metadata", color = "0.22 0.6 0.85", style="rounded"];
        b5[label = "augur export", color = "0.22 0.6 0.85", style="rounded"];
        b1 -> b2
        b1 -> b5
        b2 -> b5
        b2 -> b3
        b3 -> b5
        b4 -> b3
        b4 -> b5
    }
```

Reply:
These are great, @jameshadfield! I slightly prefer the BEAST workflow for its simplicity and its representation of an interesting use case for readers. How would you feel about a three-panel figure with A) Zika, B) TB, and C) BEAST?

Regarding aesthetics, color is basically wasted here, but it could be helpful to use one color for rules that use Augur and another color for non-Augur steps.

If we could handle most aesthetics in the current dot files would that be OK? I just want to keep this as minimal as possible.

Reply:
Here is an example of what a three panel figure with simpler colors could be:

![image](https://user-images.githubusercontent.com/85372/99119873-caaffb00-25ae-11eb-9dce-82ba777c042b.png)